bitnami/rabbitmq-cluster-operator/Chart.yaml

```
@@ -0,0 +1,26 @@
+annotations:
+  category: Infrastructure
+apiVersion: v2
+appVersion: 1.8.1
+dependencies:
+  - name: common
+    repository: https://charts.bitnami.com/bitnami
+    tags:
+      - bitnami-common
+    version: 1.x.x
+description: The RabbitMQ Cluster Kubernetes Operator automates provisioning, management, and operations of RabbitMQ clusters running on Kubernetes.
+engine: gotpl
+home: https://github.com/rabbitmq/cluster-operator
+icon: https://bitnami.com/assets/stacks/rabbitmq-cluster-operator/img/rabbitmq-cluster-operator-stack-220x234.png
+keywords:
+  - rabbitmq
+  - operator
+  - infrastructure
```

Other keywords suggestions:
```suggestion
  - rabbitmq
  - operator
  - infrastructure
  - message queue
  - AMQP
```